tools/vscode-debugging/conf/pre-debug.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p ./debug
+make deps
+make build
```

Comment:
Maybe a silly question, but why build here? I would have assume the vscode is doing some debug/repl/thing, and not using this?

Reply:
I think this was left over from the need for the extension build